operations/research-and-development/product/product-technical-writing-team-handbook.md

```diff
@@ -175,6 +185,28 @@ The Technical Product Writer/s form part of the PR review/approval process. When
 
 The [Documentation Style guide](https://docs.mattermost.com/guides/core.html#documentation-style-guide) and [UI text guidelines](https://docs.mattermost.com/process/documentation-UItext-guidelines.html).
 
+## Customer Education Responsibilities
+
+### Identifying and ranking gaps and pain points among customers
+
+When it comes to identifying gaps and pain points, ranking them is important in order to correctly prioritize.
+
+#### Identification
+
+- Monitor support tickets and get regular feedback from Support Engineers through the support channel
+- Ensure information flow with Solutions Architects and Customer Success Managers e.g. by receiving a regular report
+- Use already existing structures between Customer Support and Customer Success to avoid overhead; Customer Support is constantly sharing information with Customer Success Managers
+
+#### Ranking
+
+- Use existing metrics from the ticket system like priority and customer revenue to rank identified issues
+- Compare Customer Support time put into issues to generated revenue for high impact
+
+#### What is missing
```

Comment:
I am wondering if this is something we want to add to the handbook or whether it should rather be added when it's no longer missing and could rather be part of the AORs?

Reply:
To me it now feels like it should live in my backlog and be removed from here. Would solve [this below](https://github.com/mattermost/mattermost-handbook/pull/151#discussion_r410070847) as well. WDYT?

Reply:
My thought is that the missing components would be discussed and made into actionable points and then added to the relevant section. But, that's my opinion. @jasonblais is definitely the one to confirm this.

Reply:
@faase Makes sense, let's remove from this iteration